quize.css

```diff
@@ -0,0 +1,94 @@
+
+* {
+	font-family: sans-serif;
+	font-style: bold;
+}
+body {
+	padding: 10px;
+}
+p {
+	padding-left: 10px;
+	padding-right: 10px;
+}
+
+
+/*section 1*/
+
+.section1 {
+	border-bottom: 1px dotted gray;
```

Comment:
Style Requirements ask that "The card title has a dotted 1px border that is lightgray." Since it specifies a particular color, I'd go ahead and use that color. Plus it asks for a border all the way around, not just on the bottom.